Do not make the service name in the header a link if no `serviceUrl` is provided (#2617)
alphagov/govuk-frontend: CHANGELOG.md

## Unreleased

### Recommended changes

#### Replace deprecated `govuk-header__link--service-name` class in the header

> If you're not using the Nunjucks macros, replace any instances of the class `govuk-header__link--service-name` with `govuk-header__service-name` in the header.

Reviewer: ```suggestion
If you're not using the Nunjucks macros, replace any instances in the header of the class `govuk-header__link--service-name` with `govuk-header__service-name`.
```

> The `govuk-header__link--service-name` class has been deprecated and will be removed in a future major release.

Reviewer: ```suggestion
We've deprecated the `govuk-header__link--service-name` class, and will remove it in a future major release.
```

Reviewer: 🤣
